backend/internal/database/img.go

```diff
@@ -0,0 +1,57 @@
+package database
+
+import (
+	"context"
+
+	_ "image/jpeg"
+	_ "image/png"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+)
+
+type ImgDatabase struct {
+	Conn *pgxpool.Pool
+}
+
+/*
+ * Gets an image from the image table given the id. If image exists, returns the binary data of the image
+ */
+func (db *ImgDatabase) GetImage(ctx context.Context, id string) (*[]byte, string, error) {
+	var img []byte
+	var ctype string
+	var query string = "SELECT img, mime_type FROM images WHERE id=$1"
+
+	row := db.Conn.QueryRow(ctx, query, id)
+
+	err := row.Scan(&img, &ctype)
+
+	if err != nil {
+		// check for error message b/c sql.ErrNoRows is not the same as the error returned
+		// when image cannot be found in the database
+		if err.Error() == "no rows in result set" {
+			return nil, "", nil
+		}
+
+		return nil, "", errors.Wrap(err, "error in SELECT")
+	}
+
+	return &img, "", nil
+
```

Comment:
What is the variable ctype here? Did you mean to return it?

Reply:
If ctype isn't meant to be returned, I think we can remove the string from the return tuple

Reply:
good catch il fix it asap